autoware_launch/launch/components/tier4_simulator_component.launch.xml

```diff
@@ -2,7 +2,7 @@
 <launch>
   <arg name="launch_dummy_perception"/>
   <arg name="launch_dummy_vehicle"/>
-  <arg name="launch_dummy_localization"/>
```

Comment:
Maybe we could put description here as well, something similar to perception_mode in autoware_launch.
https://github.com/autowarefoundation/autoware_launch/blob/2244a256e75b9cbd4c728f95497aae6a8210bde1/autoware_launch/launch/autoware.launch.xml#L46

Reply:
Since this is done in https://github.com/autowarefoundation/autoware_launch/pull/1106#discussion_r1701503274. I think it's okay to keep as it is.